isis/tests/FileNameTests.cpp

```diff
@@ -74,10 +86,12 @@ TEST(FileName, Extension) {
   EXPECT_EQ("cub", file.extension());
 }
 
-//TODO Waiting for GMock
-
-// TEST(FileName, Expanded) {
-// }
+TEST(FileName, Expanded) {
+  putenv("ISISTMPROOT=/testy/mc/test/face");
+  FileName file("$ISISTMPROOT/test.cub");
+  
+  EXPECT_EQ("/testy/mc/test/face/test.cub", file.expanded());
+}
```

Comment:
Can you just use the ISISROOT environment variable here? You can just check that the expanded filename contains the ISISROOT variable and test.cub

Reply:
No because someone's ISISROOT may be set to s different directory and I don't want to override the current ISISROOT

Reply:
I mean you can just use the already set ISISROOT environment variable. That way you don't need to set another global environment variable.

Reply:
Except that ISISROOT is going to expanded differently whenever someone has it set to something else